### ISSUE TITLE
Startup.py dependes on private repo

### ISSUE THREAD

**Issue by lt72** [dotnet/perf-infra#24]
when runnign JitBench tests, the repo currently cloned seems to be davmason private fork:

`run_command('git clone -b dev https://github.com/davmason/JitBench')`

this should be changed to either the master branch of JitBench main repo, or a dedicated branch in the main repo.

**Comment by davmason** [comment]
I agree that it makes sense long term to switch to the main jitbench repo, but it's on a private one right now on purpose. The main reason is that things keep breaking and I'm trying to keep it as stable as possible.

My general plan is to switch once it's stable and producing results for a couple of weeks.

**Comment by lt72** [comment]
Great thanks
